shared/team-building/index.tsx

```diff
@@ -281,9 +304,10 @@ class TeamBuilding extends React.PureComponent<Props, {}> {
         >
           <Kb.SectionList
             ref={this.sectionListRef}
+            selectedIndex={this.props.highlightedIndex || 0}
```

Comment:
This prop does not exist in react-native `SectionList`. It's only relevant to desktop section list where entries can be selected. I don't know how to pass this prop properly.

Reply:
I added `type DesktopProps = { selectedIndex: number | undefined }` in `shared/common-adapters/section-list.d.ts` and changed exported Props type to be `React.ComponentProps<typeof SectionList> & DesktopProps`. It still resolves to `any`, but the intent is there.